pkgs/applications/virtualization/libnvidia-container/default.nix

```diff
@@ -7,6 +7,7 @@
 , libseccomp
 , rpcsvc-proto
 , libtirpc
+, linuxPackages
```

Comment:
I'm not thrilled about tying this package to a specific `linuxPackages`. Are there any alternatives? Perhaps taking `nvidia_x11` instead, and defaulting to `linuxPackages.nvidia_x11`?

Reply:
To be very honest, my nix-skills are so limited that I was happy to get this running at all. 

This is how nvidia_x11 is defined in linuxPackages:

```
    nvidiaPackages = dontRecurseIntoAttrs (callPackage ../os-specific/linux/nvidia-x11 { });
    nvidia_x11_legacy340   = nvidiaPackages.legacy_340;
    nvidia_x11_legacy390   = nvidiaPackages.legacy_390;
    nvidia_x11_beta        = nvidiaPackages.beta;
    nvidia_x11_vulkan_beta = nvidiaPackages.vulkan_beta;
    nvidia_x11             = nvidiaPackages.stable;
```
would it be possible to just write:
`, nvidia_x11 ? linuxPackages.nvidia_x11`?

Reply:
I think we want to use nvidia_x11 as an input and set it to linuxPackages.nvidia_x11 in callPackage.

Reply:
I just noticed that nvidia_x11 is passed a lot via this mechanism so I just copied it.